### PR TITLE
Update index.ts

### PR DESCRIPTION
**Pull request by fedy2**
Add supports for generic Array

### CODE REVIEW

File: test/fixtures/array-ti.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as t from "ts-interface-checker";
+// tslint:disable:object-literal-key-quotes
+
+export const IMyArrayContainer = t.iface([], {
+  "myArray": t.array("number"),
```
Comment (dsagal) [inline test/fixtures/array-ti.ts]: It would be great to add one more complex example case like `myArray2: Array<{foo: number; bar: string}>`.

**Comment by dsagal** [comment]
I think you should be able to merge it in yourself now that the PR is approved. Let me know if you'd like me to merge instead.

**Comment by fedy2** [comment]
I don't think I have the rights to merge it.

**Comment by dsagal** [comment]
That's surprising. I added another little test case, and will merge and bump version now.

**Pull request merged**